docs/reporters/notice-templates.md

```diff
@@ -1,37 +1,38 @@
-# NOTICE Templates
+# Plain text Templates
```

Comment:
Should this file also be renamed as part of this commit? Maybe `docs/reporters/notice-templates.md` -> `docs/reporters/plain-text-templates.md`? However, that would require several updates to places that link to `notice-templates.md`, too.

Reply:
The reason I kept it was because 99% of it talks about notice templates. So, I regarded the name still as a good fit.
If you want it to be renamed, no problem. Just let me know.

If it needs renaming, I could also do a follow up PR to get this one merged.

Reply:
I'm also fine with keeping the file name, but I think the heading in the file and the file name should stay aligned in either case. And as you did change the heading, I was assuming you forgot the rename.

Reply:
Ok, I've reverted the heading.